config/src/main/java/org/hyperledger/besu/config/JsonGenesisConfigOptions.java

```diff
@@ -368,6 +367,9 @@ public OptionalLong getThanosBlockNumber() {
   @Override
   public OptionalLong getLacchainPostQuantumBlockNumber() {
     return getOptionalLong("lacchainpqblock");
+  }
+
+  @Override
```

Comment:
I think it's better to keep the differences as small as possible. Even if this addition makes sense, I would leave it out b/c it doesn't add value for the Falcon precompile work